plugins/CoreConsole/Commands/CoreArchiver.php

```diff
@@ -34,32 +34,27 @@ public static function makeArchiver($url, InputInterface $input)
 
         $archiver->disableScheduledTasks = $input->getOption('disable-scheduled-tasks');
         $archiver->acceptInvalidSSLCertificate = $input->getOption("accept-invalid-ssl-certificate");
-        $archiver->shouldArchiveAllSites = (bool) $input->getOption("force-all-websites");
         $archiver->shouldStartProfiler = (bool) $input->getOption("xhprof");
         $archiver->shouldArchiveSpecifiedSites = self::getSitesListOption($input, "force-idsites");
         $archiver->shouldSkipSpecifiedSites = self::getSitesListOption($input, "skip-idsites");
-        $archiver->forceTimeoutPeriod = $input->getOption("force-timeout-for-periods");
-        $archiver->shouldArchiveAllPeriodsSince = $input->getOption("force-all-periods");
-        $archiver->restrictToDateRange = $input->getOption("force-date-range");
         $archiver->phpCliConfigurationOptions = $input->getOption("php-cli-options");
-
-        $restrictToPeriods = $input->getOption("force-periods");
-        $restrictToPeriods = explode(',', $restrictToPeriods);
-        $archiver->restrictToPeriods = array_map('trim', $restrictToPeriods);
-
         $archiver->dateLastForced = $input->getOption('force-date-last-n');
         $archiver->concurrentRequestsPerWebsite = $input->getOption('concurrent-requests-per-website');
         $archiver->maxConcurrentArchivers = $input->getOption('concurrent-archivers');
-
-        $archiver->disableSegmentsArchiving = $input->getOption('skip-all-segments');
```

Comment:
Is it possible to add this parameter again @diosmosis ?

Reply:
Oh that should still be supported by ArchiveFilter... may have forgotten some code.

Reply:
Thanks, totally missed that one